stdlib/public/core/FloatingPoint.swift

```diff
@@ -328,15 +328,24 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
   /// `infinity` is greater than this value.
   static var greatestFiniteMagnitude: Self { get }
 
-  /// The mathematical constant pi.
+  /// The [mathematical constant π][wiki], approximately equal to 3.14159.
   ///
-  /// This value should be rounded toward zero to keep user computations with
-  /// angles from inadvertently ending up in the wrong quadrant. A type that
-  /// conforms to the `FloatingPoint` protocol provides the value for `pi` at
-  /// its best possible precision.
+  /// This constant is most often defined as the numerical value of the ratio
```

Comment:
"the numerical value of the ratio ..." or "the ratio ..."?

Reply:
Oh, odd, I thought I'd changed that. How do you feel about "This constant is most often defined to equal the ratio..."?

Reply:
The inference one might draw would be that pi is sometimes defined to be equal to another value—which would be alarming.

Reply:
> pi is sometimes defined to be equal to another value

That seems like a bad interpretation to me. It's saying "**This** constant", not "**A** constant", which implies that there is only one version of it. And the summary line clearly states which constant it is, going so far as to give an approximation and a link to the Wikipedia article. I think it's pretty clear that this is talking about different ways of defining the same constant, not different values it can take. But I'm not a native speaker, so I could be wrong. Maybe a less ambiguous line would be "This constant is most often defined from the ratio..."?

Reply:
Moot now that I've removed the paragraph.